docs/howto/how-to-make-api-tool/api-tool-guide.md

```diff
@@ -0,0 +1,250 @@
+# Creating a tool with an API hit
+In this guide, we will use the [Alteryx Python SDK](https://pypi.org/project/ayx-python-sdk/) and [Alteryx Plugin CLI](https://pypi.org/project/ayx-plugin-cli/) to create a tool that pulls information from an API and find the mean, min, and max of the data.
+
+---
+## Workspace Setup
+---
+### 1. Creating a workspace
+The very first step to creating a plugin is to make a plugin workspace. We initialize a plugin workspace in an empty directory with the `sdk-workspace-init` command. Run the command and fill out the prompts, which will then start the workspace initialization process.
```

Comment:
Is there are best practice to create a workspace location on Desktop? If yes, maybe we should provide a "Tip".

Also it would be great to explain the purpose. 

"Why do we need a workspace?"